Exercise 8.js

```diff
@@ -1 +1 @@
-const discount = totalamount => totalamount > 100 ? totalamount * 0.1 : totalamount * 0.05; 
\ No newline at end of file
+const discount = totalamount => totalamount > 100 ? totalamount * 0.1 : totalamount * 0.05 
```

Comment:
Try to rename the variable using the camel case naming convention. 

For more explanations: [camel case](https://www.pluralsight.com/blog/software-development/programming-naming-conventions-explained#camel-case)